scalableminds/webknossos
# Reactify Dataset Upload View

### Mailable description of changes:
- [Dashboard.Dataset Upload] Update dataset upload form

### Steps to test:
- Goto Dashboard --> Dataset --> "Add Dataset Button"
- Upload a dataset and check that I can be successfully views. Datasets can be downloaded from the [wiki](https://github.com/scalableminds/webknossos-wiki/wiki/Test-Datasets).

- I have not idea how to test the NDstore tab

Attention before merging: This needs a braingames-lib version update for the datastore. 

### Issues:
- part of #2009 

------
